openapi.yml

```diff
@@ -0,0 +1,189 @@
+          in: path
+          required: true
+        - name: activityId
+          description: Numeric ID of the activity
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Activity"
+          description: A single activity detail
+      summary: Get a single activity detail
+  "/teams/{teamId}/members":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Member"
+          description: An array of team members
+      summary: Get list of team members
+  "/teams/{teamId}/members/{memberId}":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+        - name: memberId
+          description: Numeric ID of the member
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Member"
+          description: A single team member
+      summary: Get a single team member
+  "/teams/{teamId}/notes":
+    get:
+      parameters:
+        - name: teamId
+          description: Numeric ID of the team
+          schema:
+            type: integer
+          in: path
+          required: true
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Note"
+          description: An array of notes
+      summary: Get list of notes for the team
+components:
+  schemas:
+    Team:
+      type: object
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+    Activity:
+      type: object
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+        date:
+          format: date-time
+          type: string
+        location:
+          type: string
+    Member:
+      type: object
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        role:
+          type: string
+        email:
+          format: email
+          type: string
+        phone:
+          type: string
+    Note:
+      type: object
+      properties:

```

Comment:
The properties should correspond to the following data structure:

```
{
        "attachment_path": null, 
        "body": "Det gik rigtig godt i l\u00f8rdags :-)", 
        "created_at": "2013-01-06T23:30:38+01:00", 
        "created_by": "Ole Kristensen", 
        "title": "Kampreferat"
    }
```

Reply:
done in latest commit